land_claims.civmap.json

```diff
@@ -79,7 +83,11 @@
-{"name":"Bloom (Ashelor)","shortname":"Bloom","color":"#6c1282","declutter":1,"x":-1945,"z":-2750,"id":"a84c25e3-76eb-4b87-b1ca-f9dbdd59b892","polygon":[[[-1675,-2302],[-1647,-2327],[-1652,-2358],[-1628,-2379],[-1626,-2398],[-1601,-2402],[-1568,-2383],[-1543,-2361],[-1429,-2406],[-1380,-2371],[-1289,-2372],[-1220,-2472],[-1170,-2472],[-1077,-2514],[-1047,-2544],[-1045,-2578],[-1094,-2606],[-1116,-2609],[-1148,-2587],[-1204,-2710],[-1208,-2884],[-1382,-2881],[-1420,-2937],[-1458,-2953],[-1528,-3045],[-1428,-3145],[-1478,-3447],[-1548,-3563],[-1498,-3637],[-1454,-3671],[-1433,-3706],[-1472,-3751],[-1504,-3752],[-1544,-3729],[-1609,-3738],[-1654,-3772],[-1774,-3756],[-1828,-3729],[-1907,-3635],[-1940,-3523],[-1914,-3423],[-2060,-3249],[-2096,-3242],[-2144,-3189],[-2184,-3189],[-2224,-3224],[-2384,-3232],[-2386,-3173],[-2338,-3090],[-2231,-2980],[-2300,-2905],[-2373,-2933],[-2434,-2923],[-2495,-2858],[-2603,-2852],[-2638,-2830],[-2764,-2832],[-2800,-2726],[-2802,-2632],[-2764,-2579],[-2766,-2538],[-2704,-2476],[-2628,-2418],[-2625,-2160],[-1721,-2161],[-1703,-2277],[-1691,-2295]]]},
+{"name":"Bloom\n(Ashelor)","shortname":"Bloom","color":"#6c1282","declutter":2,"x":-1945,"z":-2750,"id":"a84c25e3-76eb-4b87-b1ca-f9dbdd59b892","polygon":[[[-1675,-2302],[-1647,-2327],[-1652,-2358],[-1628,-2379],[-1626,-2398],[-1601,-2402],[-1568,-2383],[-1543,-2361],[-1429,-2406],[-1380,-2371],[-1289,-2372],[-1220,-2472],[-1170,-2472],[-1077,-2514],[-1047,-2544],[-1045,-2578],[-1094,-2606],[-1116,-2609],[-1148,-2587],[-1204,-2710],[-1208,-2884],[-1382,-2881],[-1420,-2937],[-1458,-2953],[-1528,-3045],[-1428,-3145],[-1478,-3447],[-1548,-3563],[-1498,-3637],[-1454,-3671],[-1433,-3706],[-1472,-3751],[-1504,-3752],[-1544,-3729],[-1609,-3738],[-1654,-3772],[-1774,-3756],[-1828,-3729],[-1907,-3635],[-1940,-3523],[-1914,-3423],[-2060,-3249],[-2096,-3242],[-2144,-3189],[-2184,-3189],[-2224,-3224],[-2384,-3232],[-2386,-3173],[-2338,-3090],[-2231,-2980],[-2300,-2905],[-2373,-2933],[-2434,-2923],[-2495,-2858],[-2603,-2852],[-2638,-2830],[-2764,-2832],[-2800,-2726],[-2802,-2632],[-2764,-2579],[-2766,-2538],[-2704,-2476],[-2628,-2418],[-2625,-2160],[-1721,-2161],[-1703,-2277],[-1691,-2295]]]},
```

Comment:
e.g. Bloom I would leave as `Bloom` instead of `(Ashelor)`

... wait does ashelor actually control bloom now? or the other way around?

Reply:
Ashelor does indeed control bloom now. It was specifically requested to add that in.